kanidmd/src/lib/credential/mod.rs

```diff
@@ -214,30 +214,33 @@ impl Password {
 }
 
 #[derive(Clone, Debug)]
-pub struct BackupCode {
-    // TODO: any other attributes?
+pub struct BackupCodes {
+    // WIP_TODO: any other attributes?
     code_set: HashSet<String>,
 }
 
-//TODO: No need a separate DbBackupCodeV1 struct probably?
-impl TryFrom<DbBackupCodeV1> for BackupCode {
+//WIP_TODO: No need a separate DbBackupCodeV1 struct probably?
```

Comment:
No I think we should still have it as seperate incase we extend or change the type in the future.